database/migrations/2024_03_22_000000_add_origin_and_destination_id_to_trips.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void {
+        Schema::table('hafas_trips', static function(Blueprint $table) {
+            //Columns need to be nullable for the migration
+            $table->unsignedBigInteger('origin_id')->nullable()->after('origin');
+            $table->unsignedBigInteger('destination_id')->nullable()->after('destination');
+
+            $table->foreign('origin_id')->references('id')->on('train_stations');
+            $table->foreign('destination_id')->references('id')->on('train_stations');
```

Comment:
Bitte testen, aber das fände ich schlanker

```suggestion
            $table->foreignId('origin_id')->references('id')->on('train_stations')->nullable()->after('origin');
            $table->foreignId('destination_id')->references('id')->on('train_stations')->nullable()->after('destination');
```